### PR TITLE
Implemented issue #924

### PR DESCRIPTION
**Pull request by Soarc**
### Checklist

[+] I have included examples or tests
[+] I have updated the change log
[+] I am listed in the CONTRIBUTORS file
[+] I have cleaned up the commit history (use rebase and squash)

### Changes proposed in this pull request:

Implemented issue #924.
Fixed StyleCop issues with SharpDX projects
@oxyplot/admins

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -36,6 +36,7 @@ after_build:
   - NuGet pack Source\OxyPlot.Pdf\OxyPlot.Pdf.nuspec -version "%GitVersion_NuGetVersion%"
   - NuGet pack Source\OxyPlot.GtkSharp\OxyPlot.GtkSharp.nuspec -version "%GitVersion_NuGetVersion%"
   - NuGet pack Source\OxyPlot.GtkSharp\OxyPlot.GtkSharp3.nuspec -version "%GitVersion_NuGetVersion%"
+  - NuGet pack Source\OxyPlot.SharpDX.WPF45\OxyPlot.SharpDX.WPF45.nuspec -version "%GitVersion_NuGetVersion%"
```
Comment (objorke) [inline appveyor.yml]: I think the package name should be "OxyPlot.SharpDX.Wpf" to be consistent with "OxyPlot.Wpf". "45" should not be part of the package name.

**Comment by objorke** [comment]
I think the folder name should be the same as the namespace "OxyPlot.SharpDX.Wpf" - consistent with the older folders.

File: Source/OxyPlot.SharpDX.WPF45/D3D11Image.cs [hunk 2]
```diff
@@ -1,75 +1,90 @@
...
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using System.Windows.Interop;
-using SharpDX.Direct3D9;
-using SharpDX.Direct3D11;
-using DXGIResource = SharpDX.DXGI.Resource;
-using Device = SharpDX.Direct3D9.Device;
-using System.Windows;
+// Stolen from https://github.com/sharpdx/Toolkit/blob/master/Source/Toolkit/SharpDX.Toolkit.Game/Desktop/D3D11Image.cs
```
Comment (objorke) [inline Source/OxyPlot.SharpDX.WPF45/D3D11Image.cs]: Doesn´t the license terms say that you should not remove the license text from the original source code? SharpDX is also licensed under an MIT license and should be ok to use.
Or should we add a reference to SharpDX.Toolkit.Game?

Comment (objorke) [inline Source/OxyPlot.SharpDX.WPF45/D3D11Image.cs]: Also, remove the "Stolen from" comment when updating the attribution.

**Comment by Soarc** [comment]
Changed the package names.

**Comment by objorke** [comment]
Thanks, but please change "WPF" to "Wpf" :-)

File: Source/OxyPlot.SharpDX.WPF45/D3D11Image.cs (hunk 2 above)
Comment (Soarc) [inline Source/OxyPlot.SharpDX.WPF45/D3D11Image.cs]: I'm have a bit problems with copyrights. The original copyright is "Copyright (c) 2010-2014 SharpDX - Alexandre Mutel". So i changed header copyright tags to
<copyright file="D3D11Image.cs" company="SharpDX">
   Copyright (c) 2010-2014 SharpDX - Alexandre Mutel
</copyright>

But this gives me StyleCop Warning The company attribute in the file header must contain the following company name: OxyPlot	

and 

SA1636 : CSharp.Documentation : The copyright tag in the file header must contain the following text: Copyright (c) 2014 OxyPlot contributors.

How should i change header?

**Comment by Soarc** [comment]
Oh... This a bit harder under the windows, but i'll try.

**Comment by Soarc** [comment]
Renamed all folders and namespaces. Should be fine now.
Also added to D3D11Image.cs header original license. But it now gives StyleCop issues.

**Comment by objorke** [comment]
thank you!

**Pull request merged**